lib/iris/tests/unit/analysis/cartography/test_project.py

```diff
@@ -52,7 +52,7 @@ def setUp(self):
             1,
         )
 
-        self.tcs = iris.coord_systems.GeogCS(600000)
+        self.tcs = iris.coord_systems.GeogCS(6000000)
```

Comment:
This took a wee bit of investigation, but there has been a change in `proj` to introduce verification that difference celestial bodies are not involved in a `proj` operation i.e., the Earth and the Moon.

This test originally had markedly different `ellipses` specified from their semi-majoraxis... enough so to generate a `Source and target ellipsoid do not belong to the same celestial body` traceback from `pyproj` (`proj`).

The fix, to honour the original intent of this test, is to make the semi-majoraxis within a tolerance of each other.

See https://lists.osgeo.org/pipermail/proj/2019-March/008390.html

Reply:
Were the original tests out by a factor of 10? look like it if the value on L34 is the radius of Earth, in m.

Reply:
I think that the original semi-major axis values were just nonsense and didn't really matter at the time, just as long as they were different... but now they are different enough to cause an issue with the latest `proj`.

I've not delved into `proj` to see the exact verification check/s that are being performed, but I'm guessing that it might be some ratio based check between definition of the ellipses.

I thought I'd update the test to base it on some form of Earth reality